src/fielddef.ts

```diff
@@ -60,6 +60,17 @@ export interface FieldDef {
   title?: string;
 }
 
+/* Checks if two FieldDef are identical */
+export function identical(f1: FieldDef, f2: FieldDef): boolean {
```

Comment:
Hmm. If you want to ignore `scale`,  `axis`, `legend`, `sort` -- this might become more sensible